Windows/BuildFFmpeg.sh

```diff
@@ -36,11 +36,11 @@ cd Output/Release
 --disable-protocols \
 --disable-devices \
 --enable-protocol=file \
---enable-decoder=mp3*,pcm* \
---enable-demuxer=mp3 \
---enable-encoder=libmp3lame \
+--enable-decoder=mp3*,pcm*,hevc* \
+--enable-demuxer=mp3,hevc \
+--enable-encoder=libmp3lame,hevc* \
```

Comment:
I think we don't want HEVC encoding just yet as it involves an extra licensing obligation. If decoding/encoding are tied together though, we'll just have to let the business side know.